Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -635,7 +626,8 @@ extension SourceKitServer: MessageHandler {
 // MARK: - Build System Delegate
 
 extension SourceKitServer: BuildSystemDelegate {
-  public func buildTargetsChanged(_ changes: [BuildTargetEvent]) {
+  // FIXME: (async) Make this method isolated once `BuildSystemDelegate` has ben asyncified
```

Comment:
I don't really care since I assume you're about to remove them, but FWIW `ben` has been copy pasted to a bunch :P